website/docs/tutorials/hello-world.md

```diff
@@ -0,0 +1,111 @@
+# Building zkVM Hello World
+
+This tutorial will walk you through building your first zkVM application.
+By following the steps in this guide, you will learn how:
+
+- to use the [cargo risczero] tool to create a new project
+- to send private data to the guest program
+- zkVM executes and generates a proof of a guest program
+- guest writes public output
+
+## Step 1: Create a new project
+
+Firstly, visit [Installation] page for installing necessary software.
+Secondly, using `risczero` toolchain create a `hello-world` project from the command line. The `risczero` toolchain takes `--guest-name` parameter, a guest program you want to run on zkVM to generate a proof of its execution:
+
+```bash
+## Create a project from our starter template
+cargo risczero new hello-world --guest-name hello_guest
+cd hello-world
+```
+
+In the project folder, `hello-world`, build and run the project using `cargo run --release`.
+Use this command any time you'd like to check your progress.
+
+## Step 2 (Host): Share private data as input with the guest
+
+zkVM or a [prover] runs on the **[host]**. The host code is in `hello-world/host/src/main.rs`.
+The host creates an executor environment `ExecutorEnv` before constructing a prover.
+The host makes `input` value available to the guest before execution. It does it by adding `input` to the executor environment, which is responsible for managing guest-readable memory. When the prover executes the program, it can access input:
+
+```no_compile
```

Comment:
It'd be great if we could get test coverage for this.

Reply:
@flaub I had to put `ignore` back because `examples/Hello World` is renamed `factorials` example. I'll create a ticket to update the `hello-world` example and fix this tutorial after Istanbul